Homework_2.0.py

```diff
@@ -144,6 +123,7 @@ def add_club():
 
 @app.route("/add_fee", methods=["POST"])
 def add_fee():
+    # присваиваем возможную рыночную стоимость инстанса
     user_input = False
     try:
         item = int(request.form.get('item'))
```

Comment:
We also could imagine the following string -- '-1234' the `int` function will work but it is a very strange fee equal to negative int, just a nit